preamble.tex

```diff
@@ -38,26 +34,14 @@
 \newcommand{\pa}[1]{\paren*{#1}}
 \newcommand{\pbra}[1]{\bracken*{#1}}
 \newcommand{\pbrac}[1]{\bracen*{#1}}
+% theorem-like env setup
 
-%custom symbols
-\def\checkmark{{\tikz\fill[scale=0.4](0,.35) -- (.25,0) -- (1,.7) -- (.25,.15) -- cycle;}\hspace{0.1in}} 
-
-\title{Understanding Analysis Solutions}
-\author{Michael You}
-\date{}
-
-% Exercise theorem env
 \theoremstyle{break}
 \theorembodyfont{\upshape} % No italics in body font
-\newtheorem{exercise}{Exercise}[section]
+\newtheorem{exercise}{Exercise}[section]  % Exercise theorem env
 \newtheorem{tip}{Tip}[section]
 \newtheorem{identity}{Identity}[section]
 
-% Shortcuts for ^, ~, and programming font
-\newcommand{\tat}{{\raise.17ex\hbox{$\scriptstyle\mathtt{\sim}$}}} %tilde
-\newcommand{\tac}{\textasciicircum} %XOR
-\newcommand{\te}[1]{\texttt{#1}} %for programming font
```

Comment:
I'm not sure if this is used, but I think I'll just keep these in as my personal sty